src/helpers/common.ts

```diff
@@ -83,3 +86,10 @@ export function encParamsHexToBuf(eciesData: Omit<EciesHex, "ciphertext">): Omit
     mac: Buffer.from(eciesData.mac, "hex"),
   };
 }
+
+export function getProxyCoordinatorEndpointIndex(endpoints: string[], verifier: string, verifierId: string) {
+  const verifierIdStr = `${verifier}${verifierId}`;
+  const hashedVerifierId = keccak256(Buffer.from(verifierIdStr, "utf8"));
+  const proxyEndpointNum = new BN(hashedVerifierId, "hex").mod(new BN(endpoints.length)).toNumber();
```

Comment:
shouldn't we use umod? i guess for small lengths it's fine

Reply:
yes length of endpoint is max 5 and always less thn hashedVerifierId so it is fine.